cmd/sebak/cmd/wallet/payment.go

```diff
@@ -199,6 +207,36 @@ func makeTransactionCreateAccount(kpSource keypair.KP, kpDest keypair.KP, amount
 	return tx
 }
 
+// makeTransactionFreezing creates frozen account.
+func makeTransactionFreezing(kpSource keypair.KP, kpDest keypair.KP, amount common.Amount, seqid uint64, linked string) transaction.Transaction {
```

Comment:
Why do you make two functions ? It just just be:
```go
var fee common.Amount
 if len(linked) == 0 {
  fee = common.BaseFee
}
```

And use this when making the tx

Reply:
@Geod24 But we should also validate the fee, when tx comes.
I think it is somehow complicated if we do not use two operations.

Reply:
Yes, and it's trivial, because if the op has a `linked` account, we just don't check for fee.

Reply:
Yes but it also check the source of transaction.
For example, unfreezing payment is also no-fee.
To know the whether payment and create-account are unfreezing, we should know the source account' type(frozen account) also.

Reply:
Yes but we have that already. Check [ValidateTx](https://github.com/bosnet/sebak/blob/c19dec0e98ff67177dd13c2aae59bd810a118430/lib/node/runner/checker_ballot_transaction.go#L208)